build.gradle

```diff
@@ -17,11 +17,11 @@ buildscript {
 
 allprojects {
     // NOTE:
-    // there are three places to change the version number
-    // this file
-    // app: build.gradle (versionCode only)
-    // OsmTrack (version and versionDate)
-    project.version  "1.6.3"
+    // there are two places to change the version number
+    // - this file
+    // - app: build.gradle (versionCode only)
```

Comment:
"this file" == build.gradle so only one place?

Reply:
We need a `project.version` which is defined in the root `build.gradle` and `versionCode` which is defined in `brouter-routing-app/build.gradle`. In the current implementation version information is defined in two places, but manually adoption is needed only in `brouter-routing-app/build.gradle`.

As mentioned in the PR description it should be possible to calculate also `versionCode` automatically so only tagging would be required and not changing any file.